wherehows-web/app/components/datasets/containers/dataset-ownership.ts

```diff
@@ -108,6 +108,29 @@ export default class DatasetOwnershipContainer extends Component {
     setProperties(this, { owners, fromUpstream, upstreamUrn: datasetUrn });
   });
 
+  /**
+   * Retrieves metadata about the current ownership records including modificationTime and actor
+   * @readonly
+   * @type {(Record<'actor' | 'modificationTime', string>)}
+   * @memberof DatasetOwnershipContainer
+   */
+  @computed('owners.[]')
+  get ownershipMetadata(): Record<'actor' | 'modificationTime', string> {
+    const {
+      owners: [owner]
+    } = this;
+    const ownershipMetadata = { actor: '', modificationTime: '' };
+
+    if (owner) {
+      const { confirmedBy, modifiedTime } = owner;
```

Comment:
For owners, whenever someone adds an owner the record that is saved has `confirmedBy` == `UI` so for this situation the metadata would always show "Last saved by UI at <time>". Is that what we're going for?

Reply:
Wait what? Why was it changed to UI? It used to be the currently logged in user @cptran777

Reply:
Investigated. Looks like this is an issue on the backend. In our POST request to /owners we definitely send the current user in the `confirmedBy` field, but in the followup GET response the server returns "UI" in the `confirmedBy` field.
Maybe @alyiwang can help investigate, or I can take a look at it later to see what's up.